Fix create workspace section using chectl (#1698)
codenvy/che-docs: modules/end-user-guide/partials/proc_logging-in-to-che-server-using-cli.adoc

[id="logging-in-to-{prod-id}-using-{prod-cli}_{context}"]
= Logging in to {prod-short} using {prod-cli}

This section describes how to log in to {prod-short} using {prod-cli} tool by copying login command from {prod} Dashboard.

.Prerequisites
* A running instance of {prod}. To install an instance of {prod}, see xref:installation-guide:installing-che.adoc[].
* The {prod-short} CLI management tool. See xref:overview:using-the-chectl-management-tool.adoc[].
* {prod} Dashboard is opened in a browser.

.Procedure

> . Open user's context menu on left-bottom corner and select `Copy Login Command`

Reviewer: ```suggestion
. Open the user's context menu in the lower-left corner and select `Copy Login Command`
```

+
image::loggingin/copy_login_command_menu.png[copy_login_command_menu.png]

> . Wait for notification message appeared

Reviewer: ```suggestion
. Wait for the notification message:
```

+
image::loggingin/copy_login_command_notification.png[copy_login_command_notification.png]

> . Paste login command into a terminal and observe successful logged in

Reviewer: ```suggestion
. Paste the login command into a terminal and observe a successful login:
```
